src/depwith.py

```diff
@@ -450,6 +451,12 @@ def _crypto_withdrawal_update(crypto_withdrawal: BalanceUpdate):
             updated_records.append(ftx)
         return updated_records
     if crypto_withdrawal.status == crypto_withdrawal.STATUS_AUTHORIZED:
+
+        # we process btc onchain withdrawals as a batch so skip this
+        if crypto_withdrawal.asset == assets.BTC.symbol and crypto_withdrawal.l2_network is None:
+            logger.error('authorized BTC onchain withdrawals will be done in the batch process')
```

Comment:
is this supposed to be logger.error or logger.info?

Reply:
good catch, fixed